fearless/Modules/Staking/Analytics/Container/AnalyticsContainerViewLayout.swift

```diff
@@ -0,0 +1,61 @@
+import UIKit
+import SoraUI
+
+final class AnalyticsContainerViewLayout: UIView {
+    let segmentedControl: PlainSegmentedControl = {
+        let segmentedControl = PlainSegmentedControl()
+        segmentedControl.selectionWidth = 1
+        segmentedControl.titleColor = R.color.colorGray()!
+        segmentedControl.selectionColor = R.color.colorWhite()!
+        segmentedControl.selectedTitleColor = R.color.colorWhite()!
+        segmentedControl.titleFont = .p1Paragraph
+        return segmentedControl
+    }()
+
+    let rewardsView = AnalyticsRewardsView()
+
+    let horizontalScrollView: UIScrollView = {
+        let scrollView = UIScrollView()
+        scrollView.isPagingEnabled = true
+        scrollView.showsHorizontalScrollIndicator = false
+        scrollView.bounces = false
+        return scrollView
+    }()
+
+    let embeddedModulesStackView = UIStackView()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        backgroundColor = R.color.colorBlack()
+        setupLayout()
+    }
+
+    @available(*, unavailable)
+    required init?(coder _: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupLayout() {
+        addSubview(segmentedControl)
+        segmentedControl.snp.makeConstraints { make in
+            make.top.equalTo(safeAreaLayoutGuide)
+            make.leading.trailing.equalToSuperview()
+            make.height.equalTo(40)
+        }
+
+        addSubview(horizontalScrollView)
+        horizontalScrollView.snp.makeConstraints { make in
```

Comment:
Didn't find where do we set traling or width constraints. Also a lot of layout warnings when opening analitics screen.